Sources/SwiftCurrent_SwiftUI/Views/WorkflowItem.swift

```diff
@@ -0,0 +1,270 @@
+//
+//  WorkflowItem.swift
+//  SwiftCurrent
+//
+//  Created by Tyler Thompson on 7/20/21.
+//  Copyright © 2021 WWT and Tyler Thompson. All rights reserved.
+//
+
+import SwiftUI
+import SwiftCurrent
+
+#if (os(iOS) || os(tvOS) || targetEnvironment(macCatalyst)) && canImport(UIKit)
+import UIKit
+#endif
+
+/**
+ A concrete type used to modify a `FlowRepresentable` in a workflow.
+
+ ### Discussion
+ `WorkflowItem` gives you the ability to specify changes you'd like to apply to a specific `FlowRepresentable` when it is time to present it in a `Workflow`. You should create `WorkflowItem`s inside a `thenProceed(with:)` call on `WorkflowLauncher`.
+
+ #### Example
+ ```swift
+ WorkflowItem(FirstView.self)
+            .persistence(.removedAfterProceeding) // affects only FirstView
+            .applyModifiers {
```

Comment:
We shouldn't forget to update the documentation comments on WorkflowItem, both the example and the actual documentation of it.

Reply:
? These are updated. In both the example and here.

Reply:
Sorry, I was unclear. I was calling out this example and the documentation higher up. I see the added line at the end of the discussion section now.

This example should probably include a .thenProceed(with: to show you call that off of a WorkflowItem.

Reply:
No longer true now that we changed everything.